installer/release.sh

```diff
@@ -119,6 +119,11 @@ else
 	die "Could not generate file list"
 fi
 
+cmd_git="$(echo "$LIST" | grep '^cmd/git\.exe$')"
```

Comment:
Will this get an error if the --skip-files option is specified?  I understand that is not part of a normal build of the install.